src/pages/home/home.ts

```diff
@@ -426,7 +518,7 @@ export class HomePage {
   }
 
   showLocationsModal() {
-    const locationsModal = this.modalCtrl.create(LocationPage, {
+    let locationsModal = this.modalCtrl.create(LocationPage, {
```

Comment:
this can be a `const` since it is not being modified

Reply:
Got it.